rainforest/reporter.go

```diff
@@ -26,14 +26,22 @@ type RunStepDetails struct {
 	Browsers []RunBrowserDetails `json:"browsers"`
 }
 
+// RunTestBrowserDetails contain browsers for each run test
+type RunTestBrowserDetails struct {
+	Name   string `json:"name"`
+	Result string `json:"result"`
+}
+
 // RunTestDetails contains details about a Run Test
 type RunTestDetails struct {
-	ID        int              `json:"id"`
-	Title     string           `json:"title"`
-	CreatedAt time.Time        `json:"created_at"`
-	UpdatedAt time.Time        `json:"updated_at"`
-	Result    string           `json:"result"`
-	Steps     []RunStepDetails `json:"steps"`
+	ID            int                     `json:"id"`
+	Title         string                  `json:"title"`
+	CreatedAt     time.Time               `json:"created_at"`
+	UpdatedAt     time.Time               `json:"updated_at"`
+	Result        string                  `json:"result"`
+	Steps         []RunStepDetails        `json:"steps"`
+	HasRfaResults bool                    `json:"has_wisp_results"`
```

Comment:
Did you rename the field in the API? Otherwise we're going to be stuck with this name forever and it's going to make no sense when we come to document it in the API.

We could just add an alias in the API?

Reply:
I wanted to do it later but I do see your point, let's do it now